docs/getting-started/register-and-serve-features-training-sets.mdx

```diff
@@ -39,6 +39,19 @@ class User:
   age = ff.Feature(dataset[["user", "top_item", "timestamp"]], variant="simple", type=ff.Int, inference_store=redis)
 ```
 
+#### Registering Multiple Features
+
+You can register multiple columns from the same table. You may choose to register columns by specifying the column names in `include_columns` or register all columns except a few by specifying `exclude_columns`. You cannot include `exclude_columns` and `include_columns` in the same function call.
```

Comment:
I think the logic actually allows you to use both include and exclude at the same time. I dont see a check for it